docs/Oracle Jobs/Job Types/direct_request.md

```diff
@@ -44,16 +44,17 @@ See [shared fields](/docs/jobs/#shared-fields).
 - `$(jobRun.logTopics)`: the log's topics (`indexed` fields).
 - `$(jobRun.logData)`: the log's data (non-`indexed` fields).
 
-**Example**
+**Single Word Example**
```

Comment:
```suggestion
**Single-Word Example**
```

These sections should also use header tagging, but we can update that in a followup CL later.